qiskit/transpiler/passes/scheduling/base_scheduler.py

```diff
@@ -265,6 +265,18 @@ def _get_node_duration(
         if dag.has_calibration_for(node):
             # If node has calibration, this value should be the highest priority
             cal_key = tuple(indices), tuple(float(p) for p in node.op.params)
+            if isinstance(node.op, Reset):
+                raise RuntimeWarning(
+                    "Reset durations reported from IBM backends are currently untrustworthy. Do not rely on on scheduling output."
```

Comment:
```suggestion
                    "Qiskit scheduler assumes Reset works similarly to Measure instruction. "
                    "Actual behavior depends on the control system of your quantum backend. "
                    "Your backend may provide a plugin scheduler pass."
```

Qiskit should be backend agnostic :)